packages/ui/useScroll/useScroll.js

```diff
@@ -289,6 +290,7 @@ const useScrollBase = (props) => {
 
 	useEffect(() => {
 		mutableRef.current.resizeRegistry.parent = context;
+		mutableRef.current.resizeObserver.observe(scrollContainerRef.current);
```

Comment:
Don't we need unobserve() when unmounting?

Reply:
good point!